lib/mayaUsd/fileio/translators/translatorMesh.cpp

```diff
@@ -120,6 +120,7 @@ TranslatorMeshRead::TranslatorMeshRead(
     // timeInterval or default.
     VtVec3fArray        points;
     VtVec3fArray        normals;
+    TfToken             normals_interpolation;
```

Comment:
Very minor, but this snake_case variable does stand out in a sea of camelCase variables. Can you make it normalsInterpolation?

Reply:
sorry about that, changed and committed